app/controllers/api/v1/sessions_controller.rb

```diff
@@ -35,13 +36,18 @@ def destroy
       private
 
       def session_params
-        params.require(:session).permit(:email, :password)
+        params.require(:session).permit(:email, :password, :remember_me)
       end
 
       # Signs In the user
       def sign_in(user)
         session[:user_id] = user.id
       end
+
+      def remember(user)
+        cookie.permanent.signed[:user_id] = user.id
```

Comment:
Permanent cookies are scheduled to expire in 20 years [CHECK HERE](https://apidock.com/rails/ActionDispatch/Cookies/CookieJar/permanent).
I believe that we want the session data to stay in the client browser but not for such a long period (a couple of days may be good).